pytest_mh/conn/ssh.py

```diff
@@ -454,7 +454,10 @@ def __init__(
         # since Python will not deliver signal if the code is blocked in C
         # library. The signal is deliver only after we get back to the Python
         # code.
-        self.__conn: LibsshSession = LibsshSession(timeout=1)
+        # This must be set to a low value to allow timeout X value as
+        # shown below to work properly
+        # client.host.conn.run(..., timeout=X)
```

Comment:
I do not understand this comment followed by the previous one. Can you check it?